lib/hook.template.raw

```diff
@@ -3,12 +3,21 @@
 
 const fs = require('fs')
 const path = require('path')
+const config = getRenderedConfig()
 const ghooks = getGhooksEntryPoint()
 
 if (checkForGHooks(ghooks)) {
   require(ghooks)(__dirname, __filename)
 }
 
+function getRenderedConfig() {
+  // ghooks' actual config gets rendered here
+  // it gets substituted on ghooks' setup
+  // {{generated_config_start}}
+  return global.__testonly_ghooksConfig__ || {}
+  // {{generated_config_end}}
```

Comment:
`global.__testonly_ghooksConfig__` is atm required to thoroughly test the optional behavior to exit with an error.

The whole block from `{{generated_config_start}}` to `{{generated_config_end}}` (including their comments) gets replaced when rendering the current config into the `hook.template`.

**As a side note:**
The `hooks` also get rendered into the template. I was unsure whether to ditch them, but maybe we can make use of them at a later point.